taskvine/src/manager/vine_runtime_dir.c

```diff
@@ -163,3 +163,38 @@ void vine_set_runtime_info_path(const char *path)
 	assert(path);
 	vine_runtime_info_path = xxstrdup(path);
 }
+
+void vine_set_runtime_info_template(const char *dir)
```

Comment:
Setting the template should only modify the value from `"%Y-%m-%dT%H%M%S"`.  `vine_set_runtime_info_template` should be called inside the other function to actually create the directories.

Reply:
Correct, we are setting the variables just before the manager creation.